app/src/components/inputs/inputFilter/inputFilter.stories.jsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2017 EPAM Systems
+ *
+ *
+ * This file is part of EPAM Report Portal.
+ * https://github.com/epam/ReportPortal
+ *
+ * Report Portal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Report Portal is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Report Portal.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { storiesOf } from '@storybook/react';
+import { host } from 'storybook-host';
+import { withReadme } from 'storybook-readme';
+import {
+  EntityItemName,
+  EntityItemDescription,
+  EntityLaunchNumber,
+} from 'components/filterEntities';
+import {
+  ENTITY_NAME,
+  ENTITY_DESCRIPTION,
+  ENTITY_NUMBER,
+  CONDITION_CNT,
+  CONDITION_GREATER_EQ,
+} from 'components/filterEntities/constants';
+import { InputFilter } from './inputFilter';
+import README from './README.md';
+
+const value = 'Adam';
+const entities = [
+  {
+    id: ENTITY_NAME,
+    component: EntityItemName,
+    value: '',
```

Comment:
Please put `{}` into the `value` field to avoid console warnings in storybook.
![image](https://user-images.githubusercontent.com/31861805/51477012-300e7880-1d98-11e9-8fe5-d11b720e5ccb.png)